bluesky/run_engine.py

```diff
@@ -1818,9 +1820,10 @@ async def _read(self, msg):
 
             Msg('read', obj)
         """
-        obj = msg.obj
+        obj = check_supports(msg.obj, Readable)
         # actually _read_ the object
-        ret = obj.read(*msg.args, **msg.kwargs)
+        # TODO: deprecation warning for args and kwargs?
```

Comment:
We pass `*args, **kwargs` through in a few places to the Ability, should we?

Reply:
warn about any usage of *args and **kwargs with message about opening an issue

Reply:
https://github.com/bluesky/bluesky/pull/1502/commits/c78ba233cb2c1101f2f97d1dcd64716db8a29742